src/cnaas_nms/tools/jinja_filters.py

```diff
@@ -1,8 +1,30 @@
+"""Jinja filter functions for use in configuration templates"""
+
 import ipaddress
 import re
-from typing import Union
+from typing import Union, Optional, Callable
+
+# This global dict can be used to update the Jinja environment filters dict to include all
+# registered template filter function
+FILTERS = {}
```

Comment:
A module behaving like a Singleton is such a fun thing to (ab)use to the fullest.